apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/JpaClaimRdaSink.java

```diff
@@ -25,10 +27,12 @@
 
   private final EntityManager entityManager;
   private final Metrics metrics;
+  private final Clock clock;
 
-  public JpaClaimRdaSink(String claimType, PipelineApplicationState appState) {
+  public JpaClaimRdaSink(String claimType, PipelineApplicationState appState, Clock clock) {
     entityManager = appState.getEntityManagerFactory().createEntityManager();
     metrics = new Metrics(appState.getMetrics(), claimType);
+    this.clock = clock;
```

Comment:
It seems like quite a few classes had to be updated to pass down this variable.  I didn't realize that it would cause so many code changes.  I think this will be a useful metric, but there might be other ways to capture this information (latency that is), so if you think this is adding too much complexity, we can back it out and revisit other paths for measuring latency in the future.

Reply:
This sort of thing should be handled by a dependency manager liek Guice but the pipeline application does not use one.

Reply:
No I'm fine with the complexity.  Now that it's in place it's no big deal.  I used an explict `Clock` to facilitate testing.  It would have been possible to just call `System.currentTimeMillis()` but that would not be testable.